api/libproject/devstart.rb

```diff
@@ -34,6 +34,13 @@ def create_parser(command_name)
 
 def read_db_vars_v2(gcc)
```

Comment:
I think there are other places in here, e.g.

https://github.com/all-of-us/workbench/blob/b8abc00de57cff2a96a8ad7ef1f656c73a81eaf5/api/libproject/devstart.rb#L472

where we have hardcoded all-of-us-workbench-test. We need to fix those up too.

Reply:
Agreed, but this is just an iteration. The goal is "better," not "good."